### DISCUSSION TITLE
Wireguard: "Unable to import tunnel: invalid name"

### DISCUSSION THREAD

**Discussion by blazman** [stratosphereips/AIVPN#14] [General]
Adding Wireguard Profile on Android 12, the operation errors out with the message "Unable to import tunnel: invalid name". Per a suggestion [here ](https://www.reddit.com/r/WireGuard/comments/i74nv5/unable_to_import_tunnel_invalid_name/)I try to shorten the .conf profile filename from 29 chars to 4 chars. Wireguard then adds the same profile with the new filename succesfully.

**Comment by verovaleros** [comment]
First of all, apologies for now following back on this. We are a very small team, and the AI VPN is one of our projects which we are maintaining sporadically right now! 

I migrated this to a discussion as the issue you are mentioning originated on the Android Wireguard application. We tested on other platforms, and they did not have this issue. We discussed some time ago about this, and we were not completely on board with changing our design as a workaround to a restriction that maybe Wireguard will choose to change again later.
